src/main/webapp/assets/content/json/irc.json

```diff
@@ -351,6 +351,78 @@
       "linkedin": "charininanayakkara"
     }
   },
+  {
+    "name": "Sanjanee Waniganeththi",
+    "description": "Nuclear Physics",
+    "image": "sanjanee.jpg",
+    "social": {
+      "linkedin": "sanjanee-waniganeththi-92b3a3111"
+    }
+  },
+  {
+    "name": "Scynthya Nirukshan",
+    "description": "Bioscience Engineering",
+    "image": "scynthya.jpg",
+    "social": {
+      "linkedin": "gertrude-scynthya-nirukshan-a213476a"
+    }
+  },
+  {
+    "name": "Sumudu Herath",
+    "description": "Civil Engineering",
+    "image": "sumudu.jpg",
+    "social": {
+      "linkedin": "sumuduherath"
+    }
+  },
+  {
+    "name": "Maneesha Perera",
+    "description": "Engineering",
+    "image": "maneeshashaperera.jpg",
```

Comment:
A Small spelling mistake here. The image name is "maneeshaperera.jpg"

Reply:
```suggestion
    "image": "maneeshaperera.jpg",
```

Reply:
My apologies for getting the name wrong (again), this is sloppy. I will correct it now.
A question: I clicked the 'resolve conversation' button earlier but was wondering if that is something I should do? I assumed it was meant to indicate I was addressing the issue, but now I am thinking that maybe it has another purpose. Newbie question, sorry.

Reply:
That's alright ☺️. Usually it is marked as resolved after the requested changes are done. To indicate that you are addressing the issue, you can just put a comment.